libsplinter/src/admin/store/event.rs

```diff
@@ -32,9 +32,9 @@ pub struct AdminServiceEvent {
 /// Native representation of the `AdminServiceEvent` enum variants
 pub enum EventType {
     ProposalSubmitted,
-    ProposalVote { requester: Vec<u8> },
-    ProposalAccepted { requester: Vec<u8> },
-    ProposalRejected { requester: Vec<u8> },
+    ProposalVote { requester: messages::PublicKey },
```

Comment:
This should also be a type def here, not the messages value, since this is the native struct.

Reply:
Updated to add a native type definition for this struct